release/rllib_tests/multi_node_checkpointing_compute_config.yaml

```diff
@@ -1,8 +1,6 @@
 cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
 region: us-west-2
 
-max_workers: 3
```

Comment:
The max number of workers is already set further down so this duplicates the setting in our case.